python/tvm/relax/backend/contrib/cutlass.py

```diff
@@ -244,6 +245,14 @@ def attention_patterns():
             "cutlass.attention_bias",
             *make_attention_pattern(with_bias=True),
         ),
+        (
+            "cutlass.stacked_attention",
```

Comment:
does the order here matter? If we have a subgraph containing both reshape and attention, will `cutlass.attention` that matches only a single attention operation be selected first?

Reply:
the order matters here. I tried to change the order with stacked attention first, however, the original attention matches first.